prime-router/src/main/kotlin/Receiver.kt

```diff
@@ -253,9 +263,10 @@ open class Receiver(
             }
         }
 
-        if (topic != Topic.FULL_ELR && conditionFilter.isNotEmpty() && conditionFilter[0].isNotEmpty()) {
+        if (conditionFilter.isNotEmpty() && conditionFilter[0].isNotEmpty()) {
```

Comment:
With the new bug ticket you will need the topic check here. Maybe move this to the if in line 235